cloud-storage/onedrive/tool/src/main/webapp/WEB-INF/templates/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" 
+	"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
+	<head>
+	    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+        <link media="all" href="/library/skin/tool_base.css" rel="stylesheet" type="text/css" />
+	    <link href="/library/skin/morpheus-default/tool.css" type="text/css" rel="stylesheet" media="all" />
+        <script src="/library/js/headscripts.js" type="text/javascript"></script>
+	    <meta http-equiv="Content-Style-Type" content="text/css" />
+	</head>
+	<body class="portalBody Mrphs-portalBody">
+		<div id="Mrphs-choice" class="Mrphs-portalWrapper">
+			<fieldset>
+				<h1 th:if="${onedriveConfigured == true}" th:text="#{onedrive_configured}" class="fa fa-windows" aria-hidden="true">ONE_DRIVE_INTEGRATED</h1>
+				<h1 th:if="${onedriveConfigured == false}" class="fa fa-windows" aria-hidden="true">ONE_DRIVE_ERROR</h1>
+				<p class="buttons">
+					<span th:text="#{onedrive_button_msg}">BACK_BUTTON_MESSAGE</span>
```

Comment:
I don't understand what is going on here. Are there two buttons? Or one? Why are they different? Should one be a primary button?

Reply:
@fostersdesign there is only one button as only one of those are rendered via:
`th:if="${onedriveConfigured == true}"`
or
`th:if="${onedriveConfigured == false}"`

Reply:
https://www.thymeleaf.org/doc/tutorials/3.0/usingthymeleaf.html#conditional-expressions

Reply:
I was referring to the span and the anchor `<a>`.